client-sdk/go/testing/testing.go

```diff
@@ -83,6 +83,8 @@ var (
 	Bob = newEd25519TestKey("oasis-runtime-sdk/test-keys: bob")
 	// Charlie is the test key C.
 	Charlie = newEd25519TestKey("oasis-runtime-sdk/test-keys: charlie")
+	// Cory is the default test key used by oasis-net-runner.
+	Cory = newEd25519TestKey("ekiden test entity key seed")
```

Comment:
where is the name Cory from? It looks like we've mostly been using one name per first initial

Reply:
It is the entity key from Oasis Core, hence Cory :-)

The other idea was Debbie, the debug entity :P

Reply:
o lol

Reply:
FYI Cory is deliberately separated from Alice, Bob, Charlie convention, because it's managed by the core tests/tooling (test-runner, net-runner). But it's still useful to have it around in Oasis CLI.